test/verify/check-system-info

```diff
@@ -895,6 +895,36 @@ password=foobar
         b.wait_in_text(".pf-c-menu__item.pf-m-selected", "Custom crypto policy")
         b.click("#crypto-policy-dialog button.pf-c-button.pf-m-link")
 
+    @skipImage("crypto-policies not avaiable", "fedora-coreos", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable", "arch")
+    def testFIPSPolicy(self):
+        m = self.machine
+        b = self.browser
+        cmd = "update-crypto-policies"
+
+        # Admin sets FIPS in terminal
+        m.execute(cmd + " --set FIPS")
+        self.login_and_go("/system")
+        b.wait_text("#fips_mode_disabled_text", "FIPS crypto policy selected, but not active")
+        b.click(".system-health-crypto-policies button.pf-c-button.pf-m-link")
+        b.click("#enable-fips-mode-dialog button.pf-c-button.pf-m-primary")
+
+        # fips-mode-setup takes a while to complete.
+        m.wait_reboot(timeout_sec=180 * 3)
```

Comment:
heh -- just make it a round "600", or at least "60 * 9" (But really, 5 mins, aka. 300s ought to suffice? It should take about a minute)